src/movies.js

```diff
@@ -1,22 +1,94 @@
+const movies = require('./data');
+//console.log(movies);
+
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+
+function getAllDirectors(data) {
+  const directors = data.map((person) => {
```

Comment:
Con la arrow podemos retornar en una sola linea 😉 
```suggestion
  const directors = data.map((person) => person.director)
```